ui-grid/ui-grid.d.ts

```diff
@@ -2823,12 +2823,12 @@ declare module uiGrid {
              * returns all selected rows as gridRows
              * @returns {Array<IGridRow>} The selected rows
              */
-            getSelectedGridRows(): Array<IGridRow>;
+            getSelectedGridRows(): Array<any>;
```

Comment:
Quite right, the return value of `getSelectedGridRows()` is supposed to be `uiGrid.IGridRow`, not `any`. It is `getSelectedRows()` that returns the selected row entities. See #5684.